### PR TITLE
Fields for decision maker and date - RIP-247

### PR DESCRIPTION
**Pull request by autotelik**
https://eaflood.atlassian.net/browse/RIP-247

### CODE REVIEW

**Comment by autotelik** [comment]
:eyes:

File: app/models/flood_risk_engine/enrollment_exemption.rb [hunk 1]
```diff
@@ -20,5 +20,21 @@ def self.include_long_dredging?
...
+    def decision_at
+      return if comments.empty?
+      comments.order(:created_at).last.created_at
+    end
+
+    def decision_user_id
+      return if comments.empty?
+      comments.order(:created_at).last.user_id
+    end
+
```
Comment (reggieb) [inline app/models/flood_risk_engine/enrollment_exemption.rb]: I'd suggest adding a latest comment method that could then be used in all three methods:
```ruby
def latest_comment
  comments.order(:created_at).last
end
```

**Comment by reggieb** [comment]
I think there should be some tests of these methods.

**Comment by autotelik** [comment]
scope and tests added

**Pull request merged**